src/format.mts

```diff
@@ -23,3 +25,72 @@ export const formatOutput = async (outputPath: string) => {
 
   await Promise.all(tasks);
 };
+
+type OutputProcesser = {
+  args: (path: string) => ReadonlyArray<string>;
+  command: string;
+  name: string;
+};
+
+const formatters: Record<
+  Extract<LimitedUserConfig["format"], string>,
+  OutputProcesser
+> = {
+  biome: {
+    args: (path) => ["format", "--write", path],
+    command: "biome",
+    name: "Biome (Format)",
+  },
+  prettier: {
+    args: (path) => [
+      "--ignore-unknown",
+      path,
+      "--write",
+      "--ignore-path",
+      "./.prettierignore",
+    ],
+    command: "prettier",
+    name: "Prettier",
+  },
+};
+
+/**
+ * Map of supported linters
+ */
+const linters: Record<
+  Extract<LimitedUserConfig["lint"], string>,
+  OutputProcesser
+> = {
+  biome: {
+    args: (path) => ["lint", "--apply", path],
+    command: "biome",
+    name: "Biome (Lint)",
+  },
+  eslint: {
+    args: (path) => [path, "--fix"],
+    command: "eslint",
+    name: "ESLint",
+  },
+};
+
+export const processOutput = async ({
```

Comment:
Great work adding linting support.

I don't support putting linting and formatting in this PR.

@7nohe and I discussed dropping support in favor of letting users run the linters/formatters themselves.

If we were to keep the linters/formatters we should this implementation to lint all the generated files instead of using the built-in implementation in @hey-api so that we don't have to worry if they break compatibility or change something

Reply:
np, i'll remove